server/log.js

```diff
@@ -0,0 +1,197 @@
+'use strict';
+/**
+ * redis.js
+ *
+ * Used to store Audit logs.
+ */
+
+const Redis = require("redis");
+const logLength = 5000;
```

Comment:
What do you think about also sending the logs to Splunk? Redis could keep the most recent logs and Splunk could keep all the logs.